### PR TITLE
INTYGFV-15399: Added risk calculation and risk graphs to SRS panel

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

File: packages/webcert/src/store/srs/srsReducer.tsx [hunk 1]
```diff
@@ -10,35 +10,56 @@ import {
   updatePatientId,
   updateSrsInfo,
   updateSickLeaveChoice,
+  updateSrsQuestions,
+  updateSrsPredictions,
+  updateRiskOpinion,
+  updateUnitId,
+  updateCareProviderId,
```
Comment (SamuelTilly) [inline packages/webcert/src/store/srs/srsReducer.tsx:17]: I get error saying this action is not exported from './srsActions' when trying to build

**Review by SamuelTilly — CHANGES_REQUESTED** [review]

File: packages/webcert/src/components/srs/risk/SrsRiskGraph.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+const CustomTooltip = ({ active, payload, label }: TooltipProps<number, string>) => {
+  if (active && payload && payload.length) {
+    return <div>{getTooltipContent(label, payload)}</div>
+  }
+
+  return null
+}
+
+const getTooltipContent = (label: string, payload: any) => {
```
[the remark this answers is not in the document]
Comment (SamuelTilly) [reply packages/webcert/src/components/srs/risk/SrsRiskGraph.tsx:22]:
```typescript
import { Payload } from 'recharts/types/component/DefaultTooltipContent'

payload: Payload<string, string>[]
```

File: packages/webcert/src/components/srs/risk/SrsRiskGraph.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  getPredictionDiagnosisCode,
+  getPredictionDiagnosisDescription,
+  getRiskOpinion,
+  getSickLeaveChoice,
+  getSrsPredictions,
+} from '../../../store/srs/srsSelectors'
+import { Bar, BarChart, CartesianGrid, Cell, LabelList, LabelProps, Tooltip, TooltipProps, XAxis, YAxis } from 'recharts'
+import { getCurrentRiskDataPoint, getPreviousRiskDataPoint, getRiskDataPoint, RISK_LABELS } from '../srsUtils'
+
+const CustomTooltip = ({ active, payload, label }: TooltipProps<number, string>) => {
```
Comment (SamuelTilly) [inline packages/webcert/src/components/srs/risk/SrsRiskGraph.tsx:14]: TooltipProps<string, string>

File: packages/webcert/src/store/srs/srsReducer.tsx (hunk 1 above)
Comment (mhornfeldt) [reply packages/webcert/src/store/srs/srsReducer.tsx:17]: Now it should work

**Review by SamuelTilly — APPROVED** [review]

**Pull request merged**